contracts/vaults/UnfrozenVestingVault2.sol

```diff
@@ -0,0 +1,146 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.3;
+
+import "./VestingVault.sol";
+import "../libraries/VestingVaultStorage.sol";
+
+// You can come in but you can never leave
+contract UnfrozenVestingVault2 is AbstractVestingVault {
```

Comment:
I get that this is used for testing, but can you comment here to provide some color on what this is and why it's being used?

Reply:
I changed the name of this contract to ViewUnassigned and added some comments explaining that its only used locally for viewing unassigned tokens.